clouddriver-security/src/main/groovy/com/netflix/spinnaker/clouddriver/security/config/SecurityConfig.groovy

```diff
@@ -29,4 +31,10 @@ class SecurityConfig {
   CredentialsInitializerSynchronizable noopCredentialsInitializerSynchronizable() {
     new NoopCredentialsInitializerSynchronizable()
   }
+
+  // Allows @Value annotation to tokenize a list of strings.
+  @Bean
+  ConversionService conversionService() {
```

Comment:
could you move this into CloudDriverConfig in clouddriver-core? since its a context wide change it feels more appropriate there